_extras/guide.md

```diff
@@ -694,6 +698,7 @@ plt.show()
 
 
 
+
```

Comment:
Let's keep just one new line at the end of the file.

Reply:
There were literally 26 empty lines at the end. I have reduced this to one now.